engine/swarm/swarm_manager_locking.md

```diff
@@ -0,0 +1,153 @@
+---
+description: Automatically lock Swarm managers to protect encryption keys
+keywords:
+- swarm, manager, lock, unlock, autolock, encryption
+title: Lock your swarm to protect its encryption key
+---
+
+In Docker 1.13 and higher, the RAFT logs used by swarm managers are encrypted on
+disk by default. This at-rest encryption protects your service's configuration
+and data from attackers who gain access to the encrypted RAFT logs. One of the
+reasons this feature was introduced was in support of the new [Docker
+secrets](secrets.md) feature.
+
+When Docker restarts, the encryption key needed to encrypt and decrypt RAFT logs
+is loaded into each manager node's running memory from disk. Docker 1.13
+introduces the ability to protect the encryption key at rest by _autolocking_
```

Comment:
It also protects the TLS key (the same unlock key is used to encrypt the TLS key) that the manager node uses to connect to the rest of the swarm, because having a mTLS connection to the rest of the swarm means that you can get all the swarm data (including secrets).